lms/djangoapps/support/static/support/js/enrollment_factory.js

```diff
@@ -0,0 +1,13 @@
+;(function (define) {
+    'use strict';
+
+    define([
+        'underscore',
+        'support/js/views/enrollment'
+    ], function (_, EnrollmentView) {
+        return function (options) {
+            options = _.extend({el: $('.enrollment-content')}, options);
```

Comment:
Why is ```el``` being set as a jQuery DOM object instead of a class? By default Backbone takes selectors and creates this.$el as a DOM object anyway.